[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:35:41Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
config-server           active      1  mongodb-k8s  6/edge    54  10.152.183.113  no       Primary
mongos-k8s              active      1  mongos-k8s              0  10.152.183.160  no       
shard0                  active      1  mongodb-k8s  6/edge    54  10.152.183.87   no       Primary

Unit              Workload  Agent  Address     Ports  Message
config-server/0*  active    idle   10.1.30.77         Primary
mongos-k8s/0*     active    idle   10.1.30.72         
shard0/0*         active    idle   10.1.30.78         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:30:45 ERROR unit.config-server/0.juju-log config-server:4: Failed to add shard shard0 to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1726104644, 1), 'signature': {'hash': b'\\x1dW\\r)\\xc3\\x8bX\\xe1\\xa9\\x0eu\\xef\\x02\\xc0\\x86K>?\\xe2\\xb9', 'keyId': 7413562858014769177}}, 'operationTime': Timestamp(1726104644, 1)}")
unit-config-server-0: 01:30:45 ERROR unit.config-server/0.juju-log config-server:4: shard0 shard does not have the same auth as the config server.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...